### PR TITLE
Wizard: Prefetch the list of activation keys on Image output step

### PR DESCRIPTION
**Pull request by regexowl**
This prefetches the list of activation keys upon entering the Wizard, making the loading times on the Registration step better for the user.

### CODE REVIEW

**Comment by regexowl** [comment]
This is just a small part of activation keys prefetching, another improvement would be prefetching info about the selected activation key as well, which would required moving things between steps a bit. Which are changes that are a part of https://github.com/osbuild/image-builder-frontend/pull/2504

File: src/Components/CreateImageWizard/steps/ImageOutput/TargetEnvironment.tsx [hunk 1]
```diff
@@ -49,6 +50,11 @@ const TargetEnvironment = () => {
 
   const dispatch = useAppDispatch();
   const prefetchSources = provisioningApi.usePrefetch('getSourceList');
+  const prefetchActivationKeys = rhsmApi.usePrefetch('listActivationKeys');
+
+  useEffect(() => {
```
Comment (lucasgarfield) [inline src/Components/CreateImageWizard/steps/ImageOutput/TargetEnvironment.tsx:55]: I was trying to figure out whether we needed a useEffect or not for this. The RTKQ docs seem to use a similar pattern for this scenario so I think it makes sense!
https://redux-toolkit.js.org/rtk-query/usage/prefetching#recipe-prefetch-immediately

**Review by lucasgarfield — APPROVED** [review]
lgtm, thank you!

**Comment by ochosi** [comment]
Weeee, looking forward to this landing! 😍

**Pull request merged**